org.civicrm.reqangular/src/common/directives/timepicker-select.directive.js

```diff
@@ -25,49 +26,27 @@ define([
   function timepickerSelectController ($scope) {
     var vm = this;
 
-    vm.defaultOption = $scope.timepickerSelectDefaultOption;
+    vm.placeholder = $scope.timepickerSelectPlaceholder;
```

Comment:
Instead of doing this, please use:

```js
scope: {
  placeholder: '@timepickerSelectPlaceholder?'
}
```

**Sidenote:** I think we are missing the `?` modified for several of the bindings, which makes the binding optional. Can you please confirm?

Reply:
Why the proposed solution? If the binding changes, the placeholder will change too (devs can use this: `<select ... time-picker-select-placeholder="{{ placeholder }}"></select>`)